src/scripts/GameController.ts

```diff
@@ -85,20 +85,29 @@ class GameController {
             const keyCode = e.keyCode;
 
             if (App.game.gameState === GameConstants.GameState.dungeon) {
-                if (keyCode == 38 || keyCode == 87) {
+                if (keyCode == 38 || keyCode == 87) { // up || w
                     DungeonRunner.map.moveUp();
-                } else if (keyCode == 39 || keyCode == 68) {
+                } else if (keyCode == 39 || keyCode == 68) { // right || d
                     DungeonRunner.map.moveRight();
-                } else if (keyCode == 37 || keyCode == 65) {
+                } else if (keyCode == 37 || keyCode == 65) { // left || a
                     DungeonRunner.map.moveLeft();
-                } else if (keyCode == 40 || keyCode == 83) {
+                } else if (keyCode == 40 || keyCode == 83) { // down || s
                     DungeonRunner.map.moveDown();
-                } else if (keyCode == 32) {
+                } else if (keyCode == 32) { // space
                     DungeonRunner.openChest();
                     DungeonRunner.startBossFight();
                 }
                 e.preventDefault();
             }
+            else if (App.game.gameState === GameConstants.GameState.town) {
+                if (keyCode == 32) { // space
+                    if (player.town().gym)
+                        GymRunner.startGym(player.town().gym())
+                    else if (player.town() instanceof DungeonTown)
+                        DungeonRunner.initializeDungeon(player.town().dungeon());
+                    e.preventDefault();
```

Comment:
Please use brackets anyway

Reply:
Will do, do you want to to enable that rule for eslint too in this PR/new PR?

Reply:
Oh yes please! Different pr prefereably